php/Terminus/Commands/SiteCommand.php

```diff
@@ -1525,6 +1533,45 @@ public function setServiceLevel($args, $assoc_args) {
   }
 
   /**
+   * Enable or disable Solr indexing
+   *
+   * ## OPTIONS
+   *
+   * <enable|disable>
+   * : Options are enable and disable
+   *
+   * [--site=<site>]
+   * : Sitei on which to change Solr
+   *
+   * @subcommand solr
+   */
+  public function solr($args, $assoc_args) {
+    $action = array_shift($args);
+    $site   = $this->sites->get(Input::siteName(['args' => $assoc_args]));
+    if (in_array($site->info('service_level'), ['free', 'basic', 'pro'])) {
+      $this->failure(
+        'You must upgrade to a business or an elite plan to use Solr.'
+      );
+    }
+    switch ($action) {
+      case 'enable':
+        $redis    = $site->enableSolr();
+        if ($redis) {
+          $this->log()->info('Solr enabled. Converging bindings...');
+        }
+        $result = $site->convergeBindings();
```

Comment:
could we embed the `convergeBindings` call inside the `enableSolr` method. It seems like an implementation detail that I'd like to hide within the site object.